packages/gatsby-theme-carbon/src/components/ModifiedDate/ModifiedDate.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Row, Column } from '../Grid';
+import styles from './modified-date.module.scss';
+
+const ModifiedDate = ({ date }) => {
+  const options = {
+    day: '2-digit',
+    year: 'numeric',
+    month: 'long',
+  };
+
+  const lastModified = new Date(date);
+
+  return date ? (
+    <Row className={styles.row}>
+      <Column>
+        <div className={styles.text}>
+          Page last updated: {lastModified.toLocaleDateString('en-GB', options)}
```

Comment:
```suggestion
          Page last updated: {lastModified.toLocaleDateString(undefined, options)}
```

We don't want to force the English locale. By using `undefined` (no quotes), it will defer to the browser's setting.
![Screen Shot 2021-02-08 at 10 37 52 AM](https://user-images.githubusercontent.com/4078018/107251358-e124f180-69f9-11eb-90ff-113543128c70.png)